### ISSUE TITLE
YouTube lag

### ISSUE THREAD

**Issue by SingularityRS** [ParticleCore/Particle#306]
Browser Name: Google Chrome

Browser Version: 52.0.2743.116 m (64-bit)

Operating System:  Windows 7 Ultimate 64-bit SP1

Software Type: YouTube Plus

Software Version: 1.4.7

Problem Description: Installing this extension appears to impact how the Youtube pages respond. Problems include:

- - Scrolling occasionally causes slight pauses. It's not smooth as it is without this extension, or another YouTube-based extension.
- - If hovering my cursor over the YouTube player while its playing, the options take longer to appear (that allow me to pause the video). This doesn't happen with other YouTube extensions, or without any extension. Basically trying to interact with the page is simply noticeably slower than normal.
- Videos can take longer to play - longer than normal.

Basically, in a nutshell, it's like installing this extension creates some unwanted lag that slows things down for YouTube only. 

Steps to reproduce: Installing this extension simply re-produces the problem. Removing it returns page responsiveness to normal.

Additional details: 
It's only this extension that impacts how the YouTube pages respond. If I installed another extension, like Magic Actions for example, it doesn't affect the page. It's still as fast as it should be. I even made a new chrome user profile and it still didn't do anything.

**Comment by ParticleCore** [comment]
Have you already followed the troubleshooting guide to see whether the lag is really being caused just because of YT+ or because another extension is causing a conflict with YT+?

https://github.com/ParticleCore/Particle/wiki/Report-a-problem#troubleshooting

I have been unable to detect any sort of lag when using YT+. Some users reporting similar behavior found that the lag was being caused by a dark YouTube theme conflicting with YT+, others had a Firefox setting changed to force the flash player in YouTube which is not compatible with YT+. Please follow the troubleshooting guide or I will have to repeat the steps here again.

**Comment by ParticleCore** [comment]
I will close this for now, if you still need help please follow the instructions in my previous reply.
